spec/services/identity_linker_spec.rb

```diff
@@ -54,7 +54,7 @@
       let(:now) { Time.zone.now }
       let(:six_months_ago) { 6.months.ago }
 
-      it 'does override a previous last_consented_at by default' do
+      it 'does not override a previous last_consented_at by default' do
```

Comment:
Was this just a typo in the example name? o_O

Reply:
yes as far as I could tell